pkg/authorization/cache/policy_cache.go

```diff
@@ -18,6 +18,18 @@ import (
 	bindingregistry "github.com/openshift/origin/pkg/authorization/registry/policybinding"
 )
 
+// PolicyCacheAdapter exposes getter methods for policy indexers and reflectors for the project cache
+type PolicyCacheAdapter interface {
+	// Return refrences to the indexers for the project cache to use
+	GetPolicyBindingIndexer() *cache.Indexer
```

Comment:
Definitely need to document what these things are indexing on.  Is it configurable?